pkg/handler/payload.go

```diff
@@ -267,3 +282,48 @@ func parseImageFormDataTextToImageInputs(req *http.Request) (textToImageInput []
 		Samples:  1,
 	}}, nil
 }
+
+func parseTextFormDataTextGenerationInputs(req *http.Request) (textGeneration []triton.TextGenerationInput, err error) {
+	prompts := req.MultipartForm.Value["prompt"]
+	if len(prompts) != 1 {
+		return nil, fmt.Errorf("invalid input")
+	}
+
+	output_len := req.MultipartForm.Value["output_len"]
+	top_k := req.MultipartForm.Value["top_k"]
+	seed := req.MultipartForm.Value["seed"]
+
+	_output_len := 100
```

Comment:
Golang we do not use the first underscore, we use the camelCase convention

Reply:
done